src/SnehalW/Weekday.java

```diff
@@ -0,0 +1,43 @@
+package SnehalW;
+
+import java.util.Scanner;
+
+/*5. Write a Java program that keeps a number from the user and generates an integer between 1 and 7 and displays 
+the name of the weekday. (use switch case)*/
+
+public class Weekday {
+	static int num;
+	static String weekday;
+
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		System.out.println("Enter Weekday number between 1 to 7");
```

Comment:
what if user wanted know on 23rd date which day comes.
write logic for that
like on 23rd date Wednesdays comes..
so you have to write logic for that